device/mellanox/x86_64-mlnx_msn4800-r0/ACS-MSN4800/pg_profile_lookup.ini

```diff
@@ -0,0 +1,23 @@
+#  PG lossless profiles.
```

Comment:
normal case you should link to the file of 4700, unless you have some other reason.

Reply:
Besides, please refer [comments](https://github.com/liorghub/sonic-buildimage/pull/13#discussion_r614674045) in simx PR.

Reply:
changes on this file may be needed.
waiting for Matty Kadosh to provide more information about this issue